src/clients/Wyam/Commands/BuildCommand.cs

```diff
@@ -68,8 +68,9 @@ protected override void ParseOptions(ArgumentSyntax syntax)
             syntax.DefineOption("use-local-packages", ref _configOptions.UseLocalPackages, "Toggles the use of a local NuGet packages folder.");
             syntax.DefineOption("use-global-sources", ref _configOptions.UseGlobalSources, "Toggles the use of the global NuGet sources (default is false).");
             syntax.DefineOption("packages-path", ref _configOptions.PackagesPath, DirectoryPathFromArg, "The packages path to use (only if use-local is true).");
-            syntax.DefineOption("output-script", ref _configOptions.OutputScript, "Outputs the config script after it's been processed for further debugging.");
-            syntax.DefineOption("verify-config", ref _verifyConfig, false, "Compile the configuration but do not execute.");
+            syntax.DefineOption("output-script", ref _configOptions.OutputScript, "Outputs the config script after it's been processed for further debugging. Consider using the --ignore-config-hash directive when using this option.");
+            syntax.DefineOption("ignore-config-hash", ref _configOptions.IgnoreConfigHash, "Force evaluating the configuration file, even when no changes were detected.");
+            syntax.DefineOption("verify-config", ref _verifyConfig, false, "Compile the configuration but do not execute. Consider using the --ignore-config-hash directive when using this option.");

```

Comment:
I wonder if either/both `output-script` and/or `verify-config` should implicitly turn on `ignore-config-hash` - not sure either of those options make sense if `ignore-config-hash` is off. In that case, we should note it in the help text and throw if `ignore-config-hash` is explicitly set to `false` when using one of the other two options. Thoughts?

Reply:
I normally like explicit with cli interfaces, reduces the chance for unintended behavior - no one reads  cli docs.